### ISSUE TITLE
Fmask Error: output argument "clr_pct"...

### ISSUE THREAD

**Issue by nasavish** [nasa/dnppy#80]
Downloaded the standalone Fmask for windows and are having some issue. The package came with the following files:

![fmask](https://cloud.githubusercontent.com/assets/14100311/10771211/93b413f6-7cbd-11e5-92a3-c7d1e854114e.JPG)

After being prompted to install the MCR package, which successfully occurred, we tried running the Fmask. This is the error we received:

![fmaskerror](https://cloud.githubusercontent.com/assets/14100311/10771237/be3746e8-7cbd-11e5-818d-e54e59c85889.png)

Anyone know what is going on? The reason we are using Fmask is because it masks out cloud shadows, which the cloudmask script on dnppy does not.

**Comment by Syntaf** [comment]
Are you calling fmask from the command line? or no?

EDIT: If so, how do you call it?

**Comment by nasavish** [comment]
No. We have been trying to open the application from the file folder. I did see this thread online...

https://code.google.com/p/fmask/issues/detail?id=10

**Comment by Syntaf** [comment]
Where is your data? from a quick google search it looks like you'll need to call it from your cline

---

Say:
- Your data is in `C:/data/`
- Fmask is in `C:/tools/`

Open your command prompt

```
> cd C:/data
> C:/tools/Fmask
```

Try that

---

In the case of that issue, it was because he needed to change _drives_ as well while calling Fmask (thus the need for `\D`), I don't believe that is the case for you (unless your data is on an external harddrive)

**Comment by nasavish** [comment]
I am going to write what I understand your suggestion to mean. 
1. Change directory of file path to where data is located
2. Call file path where Fmask is located. 

I have attached images of what happened when I tried doing that. 

First image is of (1) after pressing 'enter'

![cd](https://cloud.githubusercontent.com/assets/14100311/10774040/e925364e-7ccc-11e5-8db9-110f27914e8e.JPG)

This image is of (2) before pressing 'enter'

![fmask_image](https://cloud.githubusercontent.com/assets/14100311/10774057/00c64040-7ccd-11e5-8cad-b4a01c923507.JPG)

This image is of (2) after pressing 'enter'

![fmask_error](https://cloud.githubusercontent.com/assets/14100311/10774067/0f7be54a-7ccd-11e5-81d5-d69a8c75e334.JPG)

After this step, as you can see from the image, I am receiving an error. Have I done something incorrectly? Also, I'm not sure I completely understand (2) of your instructions. Can you please elaborate?

**Comment by Syntaf** [comment]
try `Fmask.exe` , the extension is important if it's included in the name.

**Comment by nasavish** [comment]
Receiving the same error. 

However, I just realized that this computer doesn't have the hardware required to support Fmask. 

Let me give this a go on one of the other computers that is all set up.

**Comment by nasavish** [comment]
No go

**Comment by nasavish** [comment]
I tried opening the Fmask application again and, while the same error popped up as in the first post, this error followed.

![error](https://cloud.githubusercontent.com/assets/14100311/10775067/a070dcbc-7cd3-11e5-9e33-61f19a5e1551.JPG)

**Comment by Syntaf** [comment]
Is `desktop/Fmask` Fmask.exe , or the folder containing Fmask.exe? you cannot launch a folder via the command line, you need to launch an executable .

**Comment by nasavish** [comment]
It was referring to the folder. I just tried modifying it so it would launch the application but perhaps I did it incorrectly because I am getting the same error. 

![error](https://cloud.githubusercontent.com/assets/14100311/10775578/ab3325ca-7cd7-11e5-8693-306cf5dd5ee0.JPG)

**Comment by Syntaf** [comment]
That's because your command line is parsing `C:\Users\NASA DEVELOP\Desktop\Trial\Fmask\Fmask.exe` as two separate commands due to the space . Place this in `""` quotes so the cmd knows this is one single path

**Comment by Jwely** [comment]
As a note: this issue has nothing to do with dnppy, so i've added the `wontfix` label.
